### ISSUE TITLE
Modulo operator error

### ISSUE THREAD

**Issue by Sygmei** [gura-lang/gura#6]
`if (i%10==0 && 10%i) { print("It works"); }` works.
`if (i%10==0 && 10%i != 0) { print("It works"); }` works too.
`if (i%10==0 && 10%i!=0) { print("It doesn't works"); }` doesn't, it causes a SyntaxError, is there a specific reason ?

**Comment by ypsitau** [comment]
Yes, this is a bug in the token parser. I've fixed it for the next release.
